CHANGELOG.md

```diff
@@ -1,23 +1,15 @@
-# NEXT RELEASE
+# 5.8.0 Release notes
 
 ### Bugfixes
 
-* Fix a crash on some platforms when using the query parser to look for a string or binary
-  object which has a certain combination of non-printable characters.
-
-### Breaking changes
-
-* None.
+* Fix a crash on some platforms when using the query parser to look for a string
```

Comment:
@jedelbo Is there an issue # we can add?

Reply:
No, there was never an issue

Reply:
Ok, then let's just link to the PR fixing it.